tests/integration_tests/performance/test_net_tput.py

```diff
@@ -0,0 +1,290 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests the network throughput overhead added by Firecracker."""
+
+import json
+import time
+import statistics
+from subprocess import run
+import pytest
+
+import host_tools.network as net_tools
+
+RUNS = 1
+
+IPERF = 'iperf3'
+ITERATION_DURATION = 1
+WINDOW_SIZE = 10
+
+# BW in Mbits.
+BANDWIDTHS = {
+    'udp': [100, 5000, 10000],
+    'tcp': [5000, 10000, 20000, 0]
+}
+
+# Packet size used for TCP.
+PACKET_SIZE = [128, 256, 1024, 1500]
+
+# From server to client and reversed.
+MODES = ['h2g', 'g2h']
+# Corresponding MODES for iperf options.
+MODES_OPTIONS = {
+    'h2g': '-R',
+    'g2h': ''
+}
+
+PROTOCOL = ['udp', 'tcp']
+PROTOCOL_OPTIONS = {
+    'tcp': '',
+    'udp': '-u'
+}
+
+# Skip the first OMIT_TIME seconds when testing TCP
+# to avoid slow start.
+OMIT_TIME = 0
+
+PINNING_CPU = 1
+
+OFFSET_PORT = 23412
+
+OUTPUT_JSON = 'test_net_tput_{}_iperfs.json'
+
+
+class Statistics:
```

Comment:
I would move this class in a separate file inside the `framework` folder.

Reply:
I let this here for now since I am not literally using its capabilities. Probably in the future it will be extended so yes, in the next PR it will have its own file.